### PR TITLE
SSU-9: Create modal component

### PR DESCRIPTION
**Pull request by welladam**
JIRA TASK: [SSU-9](https://cheesecakelabs.atlassian.net/jira/software/c/projects/SSU/boards/414?modal=detail&selectedIssue=SSU-9)

## Behaviour
Screen Shot 2022-08-25 at 13 46 22

https://user-images.githubusercontent.com/8185973/186724690-1e8e4464-a945-4ae3-88ab-68ccf6b8b058.mov

### CODE REVIEW

**Review by vinibs — APPROVED** [review]
It's looking good! 
I noticed that, in the video, the animation seems to be in a low FPS, is it right or is it just something related to the video recording?

**Review by carolinaknoll — APPROVED** [review]
I think this is really good!

_...and I also really like that the modal is laughing 😆_

**Comment by welladam** [comment]
The video actually got worse, but it's kinda slow, I'll try to see what's going on 🤔

File: react-web/src/components/organisms/modal/index.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  className,
+  children,
+  isOpen = false,
+  setModalOpen,
+}: IModalProps): JSX.Element => {
+  return (
+    <ReactModal
+      isOpen={isOpen}
+      className={classNames(styles.modal, className)}
+      style={defaultStyles}
```
Comment (IsabellePinheiro) [inline react-web/src/components/organisms/modal/index.tsx:30]: Can we have this style inside the style file?

File: react-web/src/components/organisms/sign-transaction-modal/index.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        <Typography variant={TypographyVariant.p} text={transactionDetails} />
+      </div>
+
+      <div className={styles.buttonsContainer}>
+        <Button
+          variant={ButtonVariant.secondary}
+          onClick={onSignTransactionClick}
+          fullWidth
+        >
+          Sign Transaciton
```
Comment (IsabellePinheiro) [inline react-web/src/components/organisms/sign-transaction-modal/index.tsx:62]: Transaction*

File: react-web/src/components/organisms/modal/index.tsx (hunk 1 above)
Comment (welladam) [reply react-web/src/components/organisms/modal/index.tsx:30]: I don't think so, because the ReactModal needs `ReactModal.Styles` props 🤔

**Review by IsabellePinheiro — APPROVED** [review]

**Pull request merged**